ms/pexpireat_test.go

```diff
@@ -33,7 +34,7 @@ func TestPexpireatError(t *testing.T) {
 	}
 	k, _ := kuzzle.NewKuzzle(c, nil)
 
-	_, err := k.MemoryStorage.Pexpireat("foo", 1488540242465, nil)
+	_, err := k.MemoryStorage.Pexpireat("foo", 1524441600, nil)
```

Comment:
uh... `pexpireat` should be able to take a valid timestamp in Epoch-millis format, meaning that it should accept a `uint64` argument. If that's not the case, this must be fixed.

As for the unit test, you can pass a 64 bits magic number simply by casting it: `uint64(1488540242465)`

This remark applies for this entire PR, obviously

Reply:
@scottinet : It accepts uint64 but not on 32 bits devices obviously. It's for the i386 and armv7 Travis jobs I did this modification.

Reply:
I'm not sure to understand: uint64 accept 64 bits integers, whatever the system they are on. The problem therefore should only lie in the magic number in the unit tests, and I think that casting them to uint64 explicitly should solve the problem.

My assumption is that magic number are probably automatically casted to `int` which defaults to int32 on 32 bits system.

Reply:
By doing that we need to update Go, CGo, C++ and Java...

Reply:
But it's ok I'll do that